### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by mariam-rivera**
# Description
We structured our project with the necessary folders:
 - Navigation: Here we are going to configurate all navigation and routes for the project.
 - Utils: Here we are going to add all utilities like Dimensions, that we will need for next updates.
 - Screens: Here we are going to add the custom desgin screen like Log In, Sign In, Reservations and so on.
 - Components: Here we are going to add all components like Buttons, Inputs, and so on.
 
  ## Screenshots 
image
image

### CODE REVIEW

**Review by hearagmu — APPROVED** [review]
Good!

**Review by hearagmu — DISMISSED** [review]
Good!

**Review by JordiGN — DISMISSED** [review]
Nice! and well documented as well!

**Review by LeyaimMagma — DISMISSED** [review]

**Review by fernandoamz — COMMENTED** [review]
Be careful with your next project. This project must be created with the name in English

File: ReservacionDeVuelos/App.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Providers from './navigation';
+
+const App = () => {
+  return <Providers/>;
+}
+
+export default App;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/App.js:8]: Add new blank screen

**Review by fernandoamz — CHANGES_REQUESTED** [review]
Every file created by yourself must add a blank line at the end of the file and remove all the comments

**Review by Ing-Heber — DISMISSED** [review]
Good job, nice description!

**Review by GracielaJazmin — DISMISSED** [review]

File: ReservacionDeVuelos/App.js (hunk 1 above)
Comment (genesis9432) [reply ReservacionDeVuelos/App.js:8]: thank you for your observation, we already added the blank line at the end of the file and removed all the comments :)

**Review by ChantalRamirez — APPROVED** [review]
Good structure

**Review by ChantalRamirez — DISMISSED** [review]
Good

**Review by hearagmu — DISMISSED** [review]
Everything is fine!

**Review by ChantalRamirez — DISMISSED** [review]
The structure is good

**Review by GracielaJazmin — DISMISSED** [review]

File: ReservacionDeVuelos/src/components/FormButton.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+const FormButton = () => {
+    return(
+        <TouchableOpacity>
+            <Text>Hey, it works!</Text>
+        </TouchableOpacity>
+    );
+};
+
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/components/FormButton.js]: Review the indentation. install prettier and eslint

File: ReservacionDeVuelos/src/components/GoogleButton.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import {
+    Text,
+    TouchableOpacity
+} from 'react-native';
+
+
+const GoogleButton = () => {
+    return(
+        <TouchableOpacity>
+            <Text>Hey, it works!</Text>
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/components/GoogleButton.js]: All the strings should be in the constant file.

File: ReservacionDeVuelos/src/navigation/Routes.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const Routes = () => {
+    return(
+        // Here we will call Auth Stack
+        <TouchableOpacity>
+            <Text>Hey, it works!</Text>
+        </TouchableOpacity>
+    );
+};
+
+export default Routes;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/navigation/Routes.js]: Add new line here

File: ReservacionDeVuelos/src/navigation/index.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+// Just return providers in App.js
+
+const Providers = () => {
+    return(
+        <Routes/>
+    );
+};
+
+export default Providers;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/navigation/index.js]: same case, add a blank line here

File: ReservacionDeVuelos/src/utils/Dimensions.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import {Dimensions} from 'react-native';
+
+// Get dimensions for all the project
+export const windowWith = Dimensions.get('window').width;
+export const windowHeight = Dimensions.get('window').height;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/utils/Dimensions.js]:
```suggestion

export const { height, width } = Dimensions.get('window');

```

File: ReservacionDeVuelos/src/utils/Dimensions.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import {Dimensions} from 'react-native';
+
+// Get dimensions for all the project
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/utils/Dimensions.js]: Remove this comment

File: ReservacionDeVuelos/src/screens/Rerservation.js [hunk 8]
```diff
@@ -0,0 +1 @@
+// Screen for reservations
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/screens/Rerservation.js]: All these blank files should have the react-native basic structure

File: ReservacionDeVuelos/src/navigation/AuthStack.js [hunk 9]
```diff
@@ -0,0 +1 @@
+// Here we will create stack navigator
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/navigation/AuthStack.js]: It should have the basic structure

File: ReservacionDeVuelos/src/components/InputButton.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+const InputButton = () => {
+    return(
+        <TouchableOpacity>
+            <Text>Hey, it works!</Text>
+        </TouchableOpacity>
+    );
+};
+
+export default InputButton;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/components/InputButton.js]: Review identation

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by JordiGN — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]
Nice description section!!

**Review by tovl1 — DISMISSED** [review]
Nice job!

**Pull request closed without merging**